.github/workflows/proto-docker.yml

```diff
@@ -9,45 +9,47 @@ on:
     paths:
       - "contrib/devtools/Dockerfile"
```

Comment:
I don't think we want to push at PR changes.

Reply:
We don't. Below we don't login and push for pull requests. This is mainly to make sure it builds